app/frontends/ogc-coverages/post-coverage-rangeset.ts

```diff
@@ -20,8 +21,14 @@ export default function postCoverageRangeset(
   res: Response,
   next: NextFunction,
 ): void {
-  // copy form parameters into the query
-  req.query = req.body;
+  // merge form parameters into the query
+  const queryKeys = Object.keys(req.query);
+  const bodyKeys = Object.keys(req.body);
+  const duplicateKeys = queryKeys.filter((x) => bodyKeys.includes(x));
+  if (duplicateKeys.length) {
+    throw new RequestValidationError(`Duplicate keys "${duplicateKeys}" found from request body and query string!`);
+  }
```

Comment:
Re: the next line, note that we do use `req.query` to reverse out which URL to provide in the jobs table and possibly in other places.  I don't see any issues being caused by this, but it's something to keep an eye out for.

Reply:
@bilts , I have addressed the issues as we discussed by implementing the parameter merging in `cookies.ts`. I also moved the merging function into `util` so that it can be used by multiple places.